### ISSUE TITLE
Add united hatzala accidents filter

### ISSUE THREAD

**Issue by omerxx** [hasadna/anyway#434]
Once the `איחוד הצלה` data is integrated we would like to be able to filter it.
This would be implemented by filtering `Marker.model.get("subtype") == 21`

@LironRS  - Please add a UI option for that.

**Comment by danielhers** [comment]
Please prioritize this, I think it would be helpful.

**Comment by danielhers** [comment]
I just noticed - this added a filter for case type 1/3, but United accidents have case type 2. This should be an option (that's the whole idea of #434). Preferably a more friendly name than "1"/"2"/"3" should be shown.

**Comment by omerxx** [comment]
@LironRS Please take care of the UI and I'll hook it up

**Comment by danielhers** [comment]
@omerxx @LironRS still on this?
The "סוג תיק" filter should be expanded to allow filtering United markers.
The naming should also be more transparent, for it to be clearer that this is filtering for the data source.

**Comment by omerxx** [comment]
I just need a button.
I'll do the rest

עומר

ב-8 בפבר 2016, בשעה 21:27, Daniel Hershcovich <notifications@github.com> כתב/ה:

**Comment by danielhers** [comment]
@omerxx what is the status on this? The button is still not very informative:
![selection_005](https://cloud.githubusercontent.com/assets/263363/15626244/c1ae20fe-24c8-11e6-937f-81ce83797173.png)

Rather than 1/3 the filter should be למ"ס (which is code 1 OR 3) and איחוד הצלה (which is code 2).

**Comment by omerxx** [comment]
@danielhers I think I've started working on it but I can't recall.
Can get back with an answer in the evening unless there's someone ready to start working on it now..

**Comment by galraij** [comment]
There should still be an option to choose between 1 or 3 (סוג תיק 1, סוג תיק 3) and איחוד הצלה.

It could look something like that:

הכל
הלשכה המרכזית לסטטיסטיקה: מס׳ תיק 1/מס׳ תיק 3
איחוד הצלה

**Comment by danielhers** [comment]
Right, if someone wants to filter by file type then it should be possible, but the usual case is to filter by source (CBS/United) and it should be possible.
Some people have expressed interest in this task but they are still not done setting up the environment.

**Comment by danielhers** [comment]
@simonaho can anyone else take this?

**Comment by omerxx** [comment]
@danielhers Can I still help?
What's the status?
Is there anyone that can help with the UI?

**Comment by danielhers** [comment]
If you have time, sure, thanks!
I just saw there was no activity for a long time in this and some other issues.

**Comment by simonaho** [comment]
Hi, @omerxx great to have you back. Can you come to the meeting this evening?
I will ask someone to make the UI more informative.
Will update you when the UI is ready.

**Comment by omerxx** [comment]
@simonaho I was never away :) just had a lot of trouble lately doing anything but work
Will do my best, if not I'll try getting to this from home, didn't see the code for a while so it would probably take some more time than usual
